pmultiqc/modules/quantms/quantms.py

```diff
@@ -173,44 +173,47 @@ def __init__(self):
             self.draw_search_engine()
 
         # TODO what if multiple are found??
-        for msstats_input in self.find_log_files("quantms/msstats", filecontents=False):
-            self.msstats_input_path = os.path.join(msstats_input["root"], msstats_input["fn"])
-            self.msstats_input_valid = True
-            self.parse_msstats_input()
-
-        if config.kwargs["quantification_method"] == "spectral_counting":
-            # Add a report section with psm table plot from mzTab for spectral counting
-            self.add_section(
-                name="Peptide Spectrum Matches",
-                anchor="peptide_spectrum_matches",
-                description='This plot shows the information of peptide spectrum matches',
-                helptext='''
-                        This table shows the information of peptide spectrum matches from mzTab PSM section.
-                        ''',
-                plot=self.psm_table_html
-            )
-
-        # TODO draw protein quantification from mzTab in the future with Protein and peptide tables from mzTab
-        # currently only draw protein tabel for spectral counting
-        if not self.msstats_input_valid and config.kwargs["quantification_method"] == "spectral_counting":
-            log.warning("MSstats input file not found!")
-            self.add_section(
-                name="Protein Quantification Table",
-                anchor="protein_quant_result",
-                description='This plot shows the quantification information of proteins'
-                            ' in the final result (mainly the mzTab file).',
-                helptext='''
-                        The quantification information (Spectral Counting) of proteins is obtained from the mzTab file. 
-                        The table shows the quantitative level and distribution of proteins in different study variables and run.
-
-                        * Peptides_Number: The number of peptides for each protein.
-                        * Average Spectral Counting: Average spectral counting of each protein across all conditions with NA=0 or NA ignored.
-                        * Spectral Counting in each condition (Eg. `CT=Mixture;CN=UPS1;QY=0.1fmol`): Average spectral counting of replicates.
-
-                        Click `Show replicates` to switch to bar plots of counting in each replicate.
-                        ''',
-                plot=self.protein_quantification_table_html
-            )
+        if config.kwargs.get('disable_table', True):
```

Comment:
If the "disable" option is not found, disable? This does not make sense.

Reply:
disable_table is a flag. disable table when set disable_table flag otherwise enable

Reply:
But you set the value to True if disable_table is not found, e.g. if the flag is not there

Reply:
Maybe you are lucky that it is always there with the correct True/False value.